packages/admin/cms-admin/src/generator/future/generator.ts

```diff
@@ -28,6 +28,7 @@ export type FormFieldConfig<T> = (
 export type FormConfig<T extends { __typename?: string }> = {
     type: "form";
     gqlType: T["__typename"];
+    mode?: "edit" | "add" | "editAdd";
```

Comment:
I don't like editAdd 😬 How about `both` or `default` instead?

Reply:
You could also remove it altogether and use `mode?: "add-only" | "edit-only"`

Reply:
I also don't like editAdd.

what about `"all"`?

Reply:
> what about "all"?

Sounds good